common/static/common/templates/discussion/customwmd-prompt.underscore

```diff
@@ -0,0 +1,44 @@
+<h4 id="editor-dialog-title"><%= title %></h4>
+<div role="status" id="wmd-editor-dialog-form-errors" class="sr" tabindex="-1"></div>
+<div>
+    <form class="form">
+        <fieldset class="field-group">
+            <legend class="form-group-hd sr"><%= title %></legend>
+            <div class="field <%= uploadFieldClass %>">
```

Comment:
I was just discussing this with @robrap. The issue is that the description is itself HTML, correct? It would be good to rename the variable to make that clear, so that it is apparent that it shouldn't be escaped. There's also an interesting question as to whether that HTML is itself correctly escaped to avoid XSS. Is there any way to move the HTML down to the client, and have the server pass the raw information? It would also be good to add a comment to indicate why you had to make an exception here.